wordpress-mobile/WordPress-iOS
# Handle approve / unapprove state when swiping over a notification

Fixes #9746 

![simulator screen shot - iphone 8 - 2018-07-13 at 14 55 02](https://user-images.githubusercontent.com/2722505/42692808-2df407ae-86ad-11e8-8366-ff8236de53a4.png)

To test:
- Navigate to notifications.
- Swipe over an approved and an unapproved comment

The state (label and background color) of the leftmost button should change.
